skabase/auxiliary/utils.py

```diff
@@ -94,7 +93,6 @@ def exception_manager(cls, arguments="", callback=None):
         # cls.exception(command_name=class_name + "::" + calframe[2][3],
         #               command_inputs=str(arguments),
```

Comment:
Now that the `arguments` parameter has been removed, we might as well remove this whole commented out block.